README.md

```diff
@@ -13,57 +13,61 @@ Focused on fragmented files as used for streaming in DASH, MSS and HLS fMP4.
 
 The library has functions for parsing (called Decode) and writing (Encode) in the package `mp4ff/mp4`.
 It also contains codec specific parsing of of AVC/H.264 including complete parsing of
-SPS and PPS in the package `mp4ff.avc`. HEVC/H.265 parsing is less complete, and available as `mp4ff.hevc`.
+SPS and PPS in the package `mp4ff.avc`. HEVC/H.265 parsing is less complete, and available as `mp4ff.hevc`. The only audio codec currently supported is AAC.
 
-Traditional multiplexed non-fragmented mp4 files can also be parsed and decoded, see `examples/segment`.
+Traditional multiplexed non-fragmented mp4 files can be parsed and decoded, but the focus is on fragmented mp4 files as used in DASH, HLS, and CMAF.
 
-The focus is, however, on non-multiplexed single-track fragmented mp4 files as used in DASH, HLS, and CMAF.
+Beyond single-track fragmented files, support has been added to parse and generate multi-track
+fragmented files as can be seen in the `examples/segment` and `examples/multitrack`.
 
 The top level structure for both non-fragmented and fragmented mp4 files is `mp4.File`.
 
-In the non-fragmented files, the members Ftyp, Moov, and Mdat are used.
-A fragmented `mp4.File` file can be a single init segment, one or more media segments, or a a
-combination of both like a CMAF track which renders into a playable one-track asset.
+In a progressive (non-fragmented) `mp4.File`, the top level attributes Ftyp, Moov, and Mdat are used to point to the corresponding boxes.
 
-For fragmented files, following high-level structures are used:
+A fragmented `mp4.File` file can be a single init segment, one or more media segments, or a
+combination of both like a CMAF track which renders into a playable one-track asset.
+For fragmented files, the following high-level attributes are used:
 
-* `InitSegment` contains an `ftyp` and `moov` box and provides the metadata for a fragmented files.
+* `Init` contains an `ftyp` and `moov` box and provides the metadata for a fragmented files.
    It corresponds to a CMAF header
-* `MediaSegment` starts with an optional `styp` box and contains on or more `Fragment`s
+* `Segments` is a slice of `MediaSegment` which start with an optional `styp` box and contain on or more `Fragment`s
```

Comment:
typo: on or more -> one or more

Reply:
Thanks for checking. I checked a bit more and intstalled a grammar checker and found quite a few typos like this, but also some other text that could be clarified, so I'm pushing a commit to improve the README.md